spec/open_api_spec.rb

```diff
@@ -80,50 +87,52 @@
     end
   end
 
-  describe "Openapi schema attributes" do
-    PARSER_ATTRS = %w[readOnly nullable enum].freeze
-    context "correctly configured" do
-      ActiveRecord::Base.connection.tables.each do |table|
-        next if SKIP_TABLES.include?(table)
-
-        model_name = table.singularize.camelize
-        doc = Api::Docs["1.0"].definitions
-        doc[model_name].properties.keys.each do |attr|
-          it "The JSON Schema #{model_name} #{attr} includes valid #{PARSER_ATTRS} types for #{doc[model_name].properties[attr]}" do
-            # If there is a validator on the Activerecord model and the schema does not include the
-            #   'required' key
-            if model_name.constantize.validators_on(attr.to_sym).present? && !doc[model_name].key?("required")
-              expect(doc[model_name].properties[attr].keys & PARSER_ATTRS).not_to include(attr)
-            # If there is a 'required' key on the schema
-            elsif doc[model_name].key?("required")
-              if doc[model_name]["required"].include?(attr)
-                expect([attr] & doc[model_name]["required"]).to include(attr)
+  %w[1.0 1.1].each do |version|
```

Comment:
This diff is ugly. The only change is the addition of the minor version so all openapi spec are tested.